### PR TITLE
[387]map_scrapped_name-content

### PR DESCRIPTION
**Pull request by kanishka66**
To get these JSON files. I did following steps:
- I got some data from devtools network list
- I then copied cUrl and got it's python code from postman
- Then I ran loop for getting all id's details
- After that I converted python to json
- And finally created csv from json.

### CODE REVIEW

File: userscrap.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
+#CODE FOR GETTING DETAILS OF ONE USER
+
+# import requests
```
Comment (Lucky-Rathore) [inline userscrap.py]: remove unwanted commented code

File: userscrap.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  'content-type': 'application/json',
```
Comment (Lucky-Rathore) [inline userscrap.py:32]: we try to avoid sensitive info in code like auth token, but in this case, it's ok!

**Review by Lucky-Rathore — COMMENTED** [review]
Please check comments

File: userscrap.py (hunk 2 above)
Comment (kanishka66) [reply userscrap.py:32]: Sure. Will remove this info next time

File: userscrap.py (hunk 1 above)
Comment (kanishka66) [reply userscrap.py]: I put it as overview . But will keep only essential code next time.

**Review by Lucky-Rathore — APPROVED** [review]

**Pull request closed without merging**